spec/lib/danger/helpers/comments_helper_spec.rb

```diff
@@ -422,49 +450,4 @@ class Dummy
       expect(message).to include("Don't worry, everything is fixable.")
     end
   end
-
-  describe "comment parsing" do
-    it "detects the warning kind" do
-      expect(dummy.table_kind_from_title("1 Warning")).to eq(:warning)
-      expect(dummy.table_kind_from_title("2 Warnings")).to eq(:warning)
-    end
-
-    it "detects the error kind" do
-      expect(dummy.table_kind_from_title("1 Error")).to eq(:error)
-      expect(dummy.table_kind_from_title("2 Errors")).to eq(:error)
-    end
-
-    it "detects the warning kind" do
-      expect(dummy.table_kind_from_title("1 Message")).to eq(:message)
-      expect(dummy.table_kind_from_title("2 Messages")).to eq(:message)
-    end
```

Comment:
the above three tests are already covered in the `#table_kind_from_title` block.